core/src/gql/schema.rs

```diff
@@ -371,6 +389,7 @@ pub async fn generate_schema(
 				}
 			})
 		})
+		.description(format!("allows fetching arbitrary records"))
```

Comment:
We can simply call `to_owned()` or `to_string()` on this string. There is a lint for it. I think Clippy failed to catch it because `surrealdb_unstable` is not active when it's run.